doc/api/thaliemitter.md

```diff
@@ -0,0 +1,189 @@
+# The `ThaliEmitter` class
+
+The `ThaliEmitter` class is a bridge between the native Cordova plugin and JavaScript to give you an easy to use interface for broadcasting availability and to connect and disconnect from peer clients.
```

Comment:
Wouldn't it make sense to point out to folks that this is the class we expect people to use when they want to use the Thali local discovery and p2p capabilities with TCP/IP clients and servers without the rest of Thali? We should probably also mention that in time we will refactor this into a stand alone NPM.